auparse/interpret.c

```diff
@@ -2876,6 +2886,20 @@ int auparse_interp_adjust_type(int rtype, const char *name, const char *val)
 		(rtype == AUDIT_ADD_GROUP || rtype == AUDIT_GRP_MGMT ||
 			rtype == AUDIT_DEL_GROUP))
 		type = AUPARSE_TYPE_GID;
+	else if (rtype == AUDIT_TRUSTED_APP) {
+		if (val[0] == '"')
+			type = AUPARSE_TYPE_ESCAPED;
+		else if (strcmp(name, "id") == 0)
+			type = AUPARSE_TYPE_UID;
+		else if (is_int_string(val))
+			type = AUPARSE_TYPE_UNCLASSIFIED;
+		/* Check if we have string with only HEX symbols,
+		 * but except HEX strings like 0xABCD */
+		else if (is_hex_string(val) && (strncmp(val, "0x", 2) != 0))
```

Comment:
Starting a value with 0x has never been part of the audit coding standard for escaped fields. I don't think we should expand it to this format.

Reply:
OK, agreed.